services/core-api/app/api/mines/reports/models/mine_report_definition.py

```diff
@@ -196,6 +196,19 @@ def _apply_filters(cls, query, regulatory_authority, is_prr_only, active_ind, se
                 field_name = section_order[index]
                 filters.append(field_name.ilike(part))
 
+        if not show_expired:
+            now = datetime.now(timezone('US/Pacific'))
```

Comment:
Will this tz cause some issues? Seeing as the dates would be stored utc and the server would be running in utc

Reply:
Ah, good point.  I'll check quickly buy I suspect you're right.